### DISCUSSION TITLE
OpenAPI documentation endpoint for Ingestion Storer and Ingestion reloader

### DISCUSSION THREAD

**Discussion by elmalakomar** [mia-platform/community#483] [Q&A] [answered]
### What product are you having troubles with?

Fast Data

### What Console version are you using?

v12.0.2

### Description

Hi,
the microservices ingestion storer and ingestion reloader do expose the openAPI documentation? If so, on which endpoint?
I'm trying to reach the endpoint '/documentation/json' but I'm getting 404 error.

### Actual Outcome

Could not find the documentation

### Expected Outcome

find the documentation

**Comment by flower-of-the-bridges** [comment, marked as answer]
Ciao @elmalakomar !

Ingestion storer does not have rest APIs: its only duty is to store [ingestion messages to bucket folders](https://docs.mia-platform.eu/docs/fast_data/bucket_storage_support/overview_bucket_storage_support#ingestion-storer).

Ingestion reloader, instead, exposes APIs to [perform reingestion of specific messages/folders to a dedicated topic](https://docs.mia-platform.eu/docs/fast_data/bucket_storage_support/overview_bucket_storage_support#ingestion-reloader): however, its path is `/documentation` instead of `/documentation/json`.

So:
- for ingestion store, you can remove the documentation path
- for ingestion reloader, you can pick `/documentation`

Let me know how it goes!
